app/src/main/kotlin/com/wire/android/ui/registration/details/CreateAccountDataDetailScreen.kt

```diff
@@ -196,16 +197,14 @@ private fun AccountDetailsContent(
                     keyboardOptions = KeyboardOptions.DefaultEmailDone,
                     onKeyboardAction = { keyboardController?.hide() },
                     modifier = Modifier
-                        .padding(
-                            start = MaterialTheme.wireDimensions.spacing16x,
-                            end = MaterialTheme.wireDimensions.spacing16x,
-                            bottom = MaterialTheme.wireDimensions.spacing16x
-                        )
+                        .padding(horizontal = MaterialTheme.wireDimensions.spacing16x)
                         .testTag("emailField")
                         .focusRequester(emailFocusRequester)
                 )
-                AnimatedVisibility(visible = state.error.isEmailError()) {
-                    EmailErrorDetailText(state.error)
+                if (state.error.isEmailError()) {
+                    AnimatedVisibility(visible = state.error.isEmailError()) { EmailErrorDetailText(state.error) }
+                } else {
+                    VerticalSpace.x16()
```

Comment:
In this case `AnimatedVisibility` won't work as expected because of that if-else as it will be replaced immediately 🙃  So I would suggest to replace `AnimatedVisibility` with `Crossfade` or `AnimatedContent` and put both text and `VerticalSpace` inside it so that it's being replaced 😄 

```suggestion
                AnimatedContent(state.error.isEmailError()) { isEmailError ->
                    when (isEmailError) {
                        true -> EmailErrorDetailText(state.error)
                        false -> VerticalSpace.x16()
                    }
                }
```

Reply:
Thanks for the suggestion, applied 🚀